packages/scandipwa/src/component/MyAccountAddressBook/MyAccountAddressBook.component.js

```diff
@@ -37,7 +37,7 @@ export class MyAccountAddressBook extends PureComponent {
 
         return (
             <MyAccountAddressTable
-              title={ __('Address #%s%s', addressNumber, postfix) }
+              title={ __(`Address #${addressNumber} ${postfix}`) }
```

Comment:
This won't work, as translations may only take in static string. Something else is an issue in this case.